packages/react-native/Package.swift

```diff
@@ -9,6 +9,7 @@
 import PackageDescription
 
 let react = "React"
+let hermesVersion = "0.79.1"
```

Comment:
This is temporary, right? We would have to probably read the React Native package.json to know the version we have to build.
Also, we will probably have to make it work with a binary that is already on disk.
For example, in CI, we build hermes in a previous step and then we download in some well known location on the disk. We need a mechanism to point to that location.

Perhaps the JS script can download and manipulate the hermes tarball so that the Package.swift does not need to know details such as the version?

Reply:
This is temporary. I'm not 100% sure how we can sync the Swift package and the script - but we can do it like you suggest - have a fixed folder for the artifacts and only care about the version number from the js script.